content/en/docs/concepts/policy/resource-quotas.md

```diff
@@ -183,6 +183,154 @@ The `Terminating`, `NotTerminating`, and `NotBestEffort` scopes restrict a quota
 * `requests.cpu`
 * `requests.memory`
 
+### Resource Quota Per PriorityClass
+
+Pods can be created at a specific [priority](/docs/concepts/configuration/pod-priority-preemption/#pod-priority). If operator wants to control consumption of
+resources by pods according to the priority of the pod, this can be achieved using `scopeSelector` field in the quota spec.
+
+A quota will be matched and consumed only if `scopeSelector` in the Quota spec selects the pod.
+
+**NOTE:** `scopeSelector` is an alpha field and must be feature gate `ResourceQuotaScopeSelectors` must be enabled before using it.
```

Comment:
s/must be feature gate/the feature gate/

Reply:
please also update content/en/docs/reference/command-line-tools-reference/feature-gates.md to make sure the feature gate is listed.